### PR TITLE
Java samples update preview

### PR DESCRIPTION
**Pull request by anjgola**
First change to java code sample

### CODE REVIEW

**Comment by anjgola** [comment]
@shahabl  Please take a look.  Since the README is common we get out of updating that.

File: temperature-lcd/maven/src/main/java/iotdk/example/DisplayTemperatureOnLCD.java [hunk 1]
```diff
@@ -128,19 +128,41 @@ static void temperature_update(GroveTemp temp, GroveButton button, GroveLed led,
...
+                aPin = aPin + 512;     // A2
+                i2cPort = i2cPort + 512;  // I2C
+            }
+        } else {
+                System.err.println(unknownPlatformMessage);
+        }
+
         // button connected to D4 (digital in)
-        GroveButton button = new GroveButton(4);
+        GroveButton button = new GroveButton(dOnPin);
```
Comment (shahabl) [inline temperature-lcd/maven/src/main/java/iotdk/example/DisplayTemperatureOnLCD.java]: Should be dInPin, right?

File: pwm/maven/src/main/java/iotdk/example/PWM.java [hunk 2]
```diff
@@ -41,13 +41,28 @@
...
+    Platform platform = mraa.getPlatformType();
+    int pinNumber = 5;
+    if(platform.equals(Platform.INTEL_UP)) {
+      if(USING_GROVE_PI_SHIELD) {
+        pinNumber = pinNumber + 512; // A0 Connector (512 offset needed for the shield)
+      }
+    } else {
+        System.err.println(unknownPlatformMessage);
+    }
     // create a PWM object from MRAA using pin 6
```
Comment (shahabl) [inline pwm/maven/src/main/java/iotdk/example/PWM.java]: Do we want pin 5 or 6?

Comment (anjgola) [reply pwm/maven/src/main/java/iotdk/example/PWM.java]: 5. fixed.

File: temperature-lcd/maven/src/main/java/iotdk/example/DisplayTemperatureOnLCD.java (hunk 1 above)
Comment (anjgola) [reply temperature-lcd/maven/src/main/java/iotdk/example/DisplayTemperatureOnLCD.java]: correct. fixed.

**Pull request merged**